lib/user-interface/react/src/shared/model/model-management.model.ts

```diff
@@ -167,8 +167,8 @@ export const containerConfigSchema = z.object({
 });
 
 export const ModelRequestSchema = z.object({
-    modelId: z.string().default(''),
-    modelName: z.string().default(''),
+    modelId: z.string().min(1).default(' '),
+    modelName: z.string().min(1).default(' '),
```

Comment:
Should these have default values at all? at minimum modelId should always be defined no matter the request. UpdateModel could optionally have modelName, but Id should still be there

Reply:
This has to be done because of how we create the object for the form. I remove the ' ' when I create the form.